CHANGES.txt

```diff
@@ -122,3 +122,5 @@ Changes
 Fixes
 =====
 
+- Fixes an issue that would cause a ClassCastException to be thrown when using
```

Comment:
Maybe clarify that an error case raised a ClassCastException and that it's now returning a sensible error.

Reply:
Sure, Fixup pushed.